lib/parser.js

```diff
@@ -113,6 +113,9 @@ Parser.prototype._preprocess = function (element, context, config) {
     }
 
     let fileContent = self._fileCache[actualFilePath]; // cache the file contents to save some I/O
+    let fileBase = path.resolve(calculateNewBaseUrl(filePath, config.rootPath, config.baseUrlMap).relative);
```

Comment:
Why is `path.resolve` used here but not in `includeFile`?

Reply:
I forgot to add it in `includeFile`. Commit amended